### PR TITLE
DM-39830: Implement support for initGuiders command in OCS.

### PR DESCRIPTION
**Pull request by tribeiro**

### CODE REVIEW

File: python/lsst/ts/observatory/control/utils/roi_spec.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+
+class ROICommon(BaseModel):
+    rows: int = Field(default=50)
+    cols: int = Field(default=50)
+    integrationTimeMillis: int = Field(alias="integration_time_millis", default=100)
+
+    @validator("rows")
+    def check_rows(cls, v: int) -> int:
+        if v < 0:
```
Comment (edennihy) [inline python/lsst/ts/observatory/control/utils/roi_spec.py:40]: Is there any scenario where 0 is a valid minimum value? If not, maybe we should make these checks if v < 1.

File: python/lsst/ts/observatory/control/utils/roi_spec.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            raise ValueError(f"Minimum rows must be positive, not {v!r}.")
+        return v
+
+    @validator("cols")
+    def check_cols(cls, v: int) -> int:
+        if v < 0:
```
Comment (edennihy) [inline python/lsst/ts/observatory/control/utils/roi_spec.py:46]: Same as above.

File: python/lsst/ts/observatory/control/utils/roi_spec.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            raise ValueError(f"Minimum cols must be positive, not {v!r}.")
+        return v
+
+    @validator("integrationTimeMillis")
+    def check_integrationTimeMillis(cls, v: int) -> int:
+        if v < 0:
```
Comment (edennihy) [inline python/lsst/ts/observatory/control/utils/roi_spec.py:52]: Same as above. I think 0 here would correspond to bias..

**Review by edennihy — APPROVED** [review]
Minor comments to consider. Otherwise, approved.

File: doc/news/DM-39830.feature.rst
[the remark this answers is not in the document]
Comment (tribeiro) [reply doc/news/DM-39830.feature.rst:1]: uhm.. I am not sure, I think this is all in the category of "support init guiders".

**Pull request merged**